CMakeLists.txt

```diff
@@ -13,13 +13,22 @@ if(POLICY CMP0015)
 	cmake_policy(SET CMP0015 OLD)
 endif()
 
-include(CheckCXXCompilerFlag)
+include("cmake/HunterGate.cmake")
+HunterGate(
+    URL "https://github.com/ruslo/hunter/archive/v0.16.30.tar.gz"
+    SHA1 "c5b159b48a3a842697f08fe2a72464adfd78a55d"
+    )
 
+project(YAML_CPP)
 
 ###
 ### Project settings
 ###
-project(YAML_CPP)
+include(CheckCXXCompilerFlag)
+hunter_add_package(Boost)
+find_package(Boost CONFIG REQUIRED)
```

Comment:
Sorry. I wrote the `find_package` in the code here. The original file also contains it at line 125. And after it adds Boost include dir to the include directories. I erase it from here.